modules/tensor_mechanics/test/tests/crystal_plasticity/stress_update_material_based/use_substep_dt.i

```diff
@@ -0,0 +1,222 @@
+[GlobalParams]
+  displacements = 'ux uy uz'
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 4
+  ny = 4
+  nz = 4
+  elem_type = HEX8
+  displacements = 'ux uy uz'
+[]
+
+# [Variables]
+#   [./ux]
+#   [../]
+#   [./uy]
+#   [../]
+#   [./uz]
+#   [../]
+# []
```

Comment:
Might as well remove these commented out variables to clean up the input file a bit